rasa/data.py

```diff
@@ -156,21 +154,10 @@ def _is_valid_filetype(path: Text) -> bool:
 
 
 def _is_nlu_file(file_path: Text) -> bool:
-    with open(file_path, encoding="utf-8") as f:
-        if file_path.endswith(".json"):
-            content = io_utils.read_json_file(file_path)
-            is_nlu_file = (
-                isinstance(content, dict) and content.get("rasa_nlu_data") is not None
-            )
-        else:
-            is_nlu_file = any(_contains_nlu_pattern(l) for l in f)
-    return is_nlu_file
-
-
-def _contains_nlu_pattern(text: Text) -> bool:
-    nlu_pattern = r"\s*##\s*(intent|regex||synonym|lookup):"
+    """Checks whether a file is an NLU file."""
+    from rasa.nlu.training_data import loading
```

Comment:
Can we move that out again and do a top level import, so that it's not imported for every file? To then have working imports you just have to move the import ` from rasa import data` in the `domain` module to `from_directory`